src/main/java/com/labsynch/labseer/exceptions/DupeLotException.java

```diff
@@ -1,6 +1,6 @@
 package com.labsynch.labseer.exceptions;
 
-public class DupeLotException extends Exception {
+public class DupeLotException extends RuntimeException {
```

Comment:
This was necessary to make sure the transaction failed.  Without a runtime exception, transactions don't roll back on thrown errors.